cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/CruiseControlApiImpl.java

```diff
@@ -65,7 +65,7 @@ public HttpClientOptions getHttpClientOptions() {
         }
     }
 
-    private static HTTPHeader generateAuthHttpHeader(String user, String password) {
+    public static HTTPHeader generateAuthHttpHeader(String user, String password) {
```

Comment:
Does it really need to be public? Wouldn't some lower level be sufficient? If it hs to be public, please add Javadoc.